WalletWasabi/Blockchain/TransactionBuilding/BnB/CoinSelection.cs

```diff
@@ -0,0 +1,38 @@
+using System.Linq;
+
+namespace WalletWasabi.Blockchain.TransactionBuilding.BnB;
+
+/// <summary>
+/// Models a coin selection with <see cref="PaymentAmount">sum of coin effective values</see> and
+/// <see cref="TotalCosts">total spending sum for the payer</see>.
+/// </summary>
+public class CoinSelection
```

Comment:
note: Introduced to reduce the number of fields in `SelectionStrategy.cs`. Also the API of `CoinSelection` is designed so that it's hard to *not assign fields at once*